vcd/resource_vcd_vapp_vm.go

```diff
@@ -497,6 +497,24 @@ var vappVmSchema = map[string]*schema.Schema{
 			},
 		},
 	},
+	"cpu_hot_add_enabled": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		Description: "True if the virtual machine supports addition of virtual CPUs while powered on.",
+	},
+	"memory_hot_add_enabled": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		Description: "True if the virtual machine supports addition of memory while powered on.",
+	},
+	"prevent_reboot": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		Description: "True if the update of resource should fail when virtual machine reboot needed.",
```

Comment:
Is this description coming from API? If I read it without context, first question comes to mind - "is it any update or specific to some fields"?

Reply:
descriptions is mine